lib/game.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
```

Comment:
SRP: This class feels like it has two responsibilities: running a game and also getting/translating input.